templates/aws-tap-entrypoint-existing-vpc.template.yaml

```diff
@@ -2328,230 +2328,152 @@ Resources:
         Fn::Base64: !Sub
           - |
             #!/bin/bash
-            set -xe
-            user=ubuntu
-            arch=$(dpkg --print-architecture)
-            uname=$(uname -m)
-            tap_dir=/home/$user/tap-setup-scripts
+            set -e
```

Comment:
Now, this is the right time to move the entire bash script into a cloud-init file. so that we keep only AWS related stuff in template/*.yaml files.

Reply:
I'd rather do that in a separate junk.

We can't just externalize everything right now, because we still need to install stuff first, most notably the `aws` CLI, to even be able to pull in the externalized could-init script.
How I roughly split it right now is:
- install things, upgrade, ... inline
- setup cfn-signal trap ... inline
- pull and exec externalized cloud-init, which mostly handles TAP things

In this PR my main concern is cfn-signal, the only reason why I externalized some of the user-data script is solely because user-data grew too big with the long HandelURLs.

I however agree that there is general opportunity to make our setup a bit better when it comes to cloud-init and passing in config/data ; I have pulled that out in TAPPC-163 for further discussions and prioritizing.